lib/responses.js

```diff
@@ -202,7 +210,12 @@ function notAcceptable(options) {
  */
 function denyTrailingSlashOnId(req, res, next) {
   if (_.last(req.path) === '/') {
-    sendDefaultResponseForCode(400, 'Trailing slash on RESTful id in URL is not acceptable', res);
+    const code = 400,
+      message = 'Trailing slash on RESTful id in URL is not acceptable';
+
+    log('error', message, { code });
+
+    sendDefaultResponseForCode(code, message, res);
```

Comment:
(see +171 to +173)

Reply:
the context for the 400 from this function and the one on line 235 are different so it would be worth logging it